### PR TITLE
epylint: corrects msg-template object

### LINKED ISSUE #1269: msg-template not editable when pylint is called from a python script
### Steps to reproduce
Call pylint from a python script as:
(pylint_stdout, pylint_stderr) = lint.py_run(command_options = "%s -E --msg-template={path} {msg} my_module.py", return_std=True)
and print the output of pylint_stdout.readlines()

### Current behavior
When printed, pylint_stdout is not formatted as defined in command_options (see above), but instead, the message is shown as defined in pylint/eplyint.py (line 90-92):

cmd = [sys.executable, "-c", run_cmd] + options + [
    '--msg-template', '{path}:{line}: {category} ({msg_id}, {symbol}, {obj}) {msg}',
    '-r', 'n', child_path] 

### Expected behavior
The output when printing pylint_stdout.readlines() should be: 
PATH MSG

But you get, instead:
PATH:188: error (E1101, no-member, OBJECT) MESSAGE

### pylint --version output
No config file found, using default configuration
pylint 1.6.4, 
astroid 1.4.8
Python 2.7.6 (default, Oct 26 2016, 20:30:19) 
[GCC 4.8.4]

### PR DESCRIPTION
**Pull request by DnPlas**
The way this object is defined won't let the user modify the message
template when pylint is called from a python script. Rearranging the
order of variables on its definition prevents this behavior.

Fixes #1269

Signed-off-by: Daniela Plascencia <daniela.plascencia@linux.intel.com>

### Fixes / new features
- Fixes #1269

### CODE REVIEW

**Comment by rogalski** [comment]
@DnPlas I restarted Travis build and (as expected) everything completed successfully.

Do you mind adding relevant entries to Changelog and What's new files? Having "Fixes #1269" somewhere in commit message would also be nice.

File: ChangeLog [hunk 1]
```diff
@@ -279,6 +279,13 @@ Release date: tba
 
       Fixes #1031
 
+What's new in Pylint 1.6.4?
+===========================
+
+Release date: 2016-07-19
```
Comment (PCManticore) [inline ChangeLog]: This should go in the 2.0 section, not here.

File: ChangeLog [hunk 2]
```diff
@@ -279,6 +279,13 @@ Release date: tba
...
+
+    * Lets the user modify msg-template when Pylint is called from a Python script
```
Comment (PCManticore) [inline ChangeLog]: Let*

File: ChangeLog [hunk 3]
```diff
@@ -279,6 +279,8 @@ Release date: tba
 
       Fixes #1031
 
+    * Let the user modify msg-template when Pylint is called from a Python script
+      Fixes #1269
```
Comment (rogalski) [inline ChangeLog]: It needs one extra empty line between actual message and comment. See other entries in ChangeLog file for reference.

Comment (DnPlas) [inline ChangeLog]: Okay, adding the extra line

**Pull request merged**